administrator/components/com_admin/sql/updates/postgresql/4.0.0-2020-09-19.sql

```diff
@@ -0,0 +1 @@
+UPDATE "#__menu" SET "link"='index.php?option=com_finder' WHERE "id"=13;
```

Comment:
```suggestion
UPDATE "#__menu" SET "link"='index.php?option=com_finder' WHERE "menutype"='main' AND "title"='com_finder' AND "link"='index.php?option=com_finder&view=index';
```
Just to save you some work so you can apply it with one click.